fastlane/lib/fastlane/helper/sh_helper.rb

```diff
@@ -54,7 +54,7 @@ def self.sh_control_output(*command, print_command: true, print_command_output:
         Open3.popen2e(*command) do |stdin, io, thread|
           io.sync = true
           io.each do |line|
-            UI.command_output(line.strip) if print_command_output
+            UI.command_output(line) if print_command_output
```

Comment:
I'm a little concerned that removing this `.strip` will make the output a bit harder to read / uglier? 🤔 Is this causing actual issues or just making the output harder to read (for certain commands)?

Reply:
It is changing the output from what the programs originally output - so it is currently actually making the output of some commands harder to read. 

Take the "Whitespace" example from https://github.com/fastlane/fastlane/issues/15021#issue-467502439 The original output (second code block) has clear hierarchy, the changed output (first code block) looses that completely. Same for the missing indentation in the "Empty Lines" example.